rust/agama-cli/src/auth.rs

```diff
@@ -109,6 +121,10 @@ impl Credentials for MissingCredentials {
 fn jwt_file() -> Option<PathBuf> {
     Some(home::home_dir()?.join(DEFAULT_JWT_FILE))
 }
+/// Path to agama-live token file.
+fn agama_token_file() -> Option<PathBuf> {
+    home::home_dir().map(|p| p.join(DEFAULT_AGAMA_TOKEN_FILE))
```

Comment:
There is nothing like a default Agama token file. Actually, the `serve` token allows to pass a path through the `--generate-token` option.

Reply:
we could ignore it then, I just followed the PBI suggestions which mentioned the /run/agama/token file

Reply:
As said, we can leave it as it is. Benefits? Well, 1) encapsultation and 2) defining our own simplified API). We follow a similar approach in the JavaScript side. But let's keep it as it is until the benefits become more clear.

Reply:
For the live media, I guess we will inject that token into root's home directory. Or we can decide that we are going to use always that file (we could even have a AGAMA_RUN_DIR environment variable).